### PR TITLE
Better layer dirnames

### PR DESCRIPTION
**Pull request by MattF-NSIDC**

### CODE REVIEW

File: qgreenland/tasks/common/raster.py [hunk 1]
```diff
@@ -108,6 +117,35 @@ def run(self):
             )
 
 
+class ReformatRaster(LayerTask):
```
Comment (trey-stafford) [inline qgreenland/tasks/common/raster.py:120]: Worth renaming this to `GdalTranslate`?

File: qgreenland/tasks/layers/dms_gtk_topo.py [hunk 2]
```diff
@@ -16,7 +17,11 @@ class DmsGtkTopo(LayerPipeline):
 
     # TODO: Build overviews for performance
```
Comment (trey-stafford) [inline qgreenland/tasks/layers/dms_gtk_topo.py:18]: OBE?

**Review by trey-stafford — APPROVED** [review]

File: qgreenland/tasks/layers/dms_gtk_topo.py (hunk 2 above)
Comment (MattF-NSIDC) [reply qgreenland/tasks/layers/dms_gtk_topo.py]: 8cbf29a

File: qgreenland/tasks/common/raster.py (hunk 1 above)
Comment (MattF-NSIDC) [reply qgreenland/tasks/common/raster.py]: 8cbf29a

**Pull request merged**